src/util/Filter.js

```diff
@@ -210,6 +217,14 @@ function simplifyArg(arg, category) {
     case 'mana':
       res = parseManaCost(arg)
       break;
+    case 'rarity':
+      let argLower = arg.toLowerCase()
+      if (rarityMap.has(argLower)) {
+        res = rarityMap.get(argLower);
+      } else {
+        res = arg;
+      }
```

Comment:
I think this entire case could be simplified as:
```suggestion
res = rarityMap[arg.toLowerCase()] ? rarityMap[arg.toLowerCase()] : arg;
```

Reply:
I was trying to match the style of the block above for colors and wasn't sure on ternary operators. Would we want to change that block to this style as well? Possibly introduce a utility function for `getOrDefault` as well?

Reply:
Yeah ternary operators are great if they are clean, like this example

Reply:
Updated with ternary and combined the mapping cases to make future additions easier.

Reply:
I don't like adding the extra map, if it doesn't eliminate the switch statement. Can we also not use .get() and .has() to maintain a consistent style?

Reply:
Indexing doesn't work correctly with Map objects, at least in my testing on Chrome so we do need to use get. I did make it a little shorter and removed the extra map in the next commit.